### PR TITLE
chore(ButtonView): add iconLiga

### PR DESCRIPTION
**Pull request by wegry**
Allow users to specify a default ButtonView.innerText for use with iconClass and icon fonts

### CODE REVIEW

File: src/platform-implementation-js/dom-driver/gmail/widgets/buttons/button-view.ts [hunk 1]
```diff
@@ -41,11 +43,12 @@ export default class ButtonView implements ButtonViewI {
   private _eventStream: Bus<any, any>;
 
   constructor(options: ButtonViewOptions) {
+    const { iconLigaDefault, iconClass } = options;
     this._hasDropdown = false;
     this._isEnabled = options.enabled !== false;
 
-    this._iconClass = options.iconClass;
-    this._iconUrl = options.iconUrl;
```
Comment (ruthhalteman) [inline src/platform-implementation-js/dom-driver/gmail/widgets/buttons/button-view.ts:48]: why is this line removed, should it be there in addition not instead?

Comment (wegry) [reply src/platform-implementation-js/dom-driver/gmail/widgets/buttons/button-view.ts:48]: Good catch. See [3f42061](https://github.com/InboxSDK/InboxSDK/commit/3f420617fcc7285be1bdbb93c51de7c6841cacec)

File: src/inboxsdk.d.ts [hunk 2]
```diff
@@ -415,6 +415,13 @@ export interface DrawerDescriptor {
 export { type MoleOptions as MoleDescriptor };
 export interface ButtonDescriptor {
   title?: string;
+  /**
+   * The innerText of the button.
+   * If you want to use an icon font with ligatures enabled, you can use the `iconLigaDefault` property.
+   *
+   * @default '&nbsp;'
+   */
+  iconLigaDefault?: string;
```
Comment (ruthhalteman) [inline src/inboxsdk.d.ts:424]: what's the reasoning for having "default" be part of this property name? it's not really a default, it's just setting something, so it was a bit confusing to me

Comment (wegry) [reply src/inboxsdk.d.ts:424]: I'm not attached to the name by any means. Would this be more your speed?

```suggestion
  iconLiga?: string;
```

The TSDoc could be clearer that we only set this property if `iconClass` is also passed.

File: src/inboxsdk.d.ts [hunk 3]
```diff
@@ -415,6 +415,13 @@ export interface DrawerDescriptor {
 export { type MoleOptions as MoleDescriptor };
 export interface ButtonDescriptor {
   title?: string;
+  /**
+   * The innerText of the button.
+   * If you want to use an icon font with ligatures enabled, you can use the `iconLigaDefault` property.
```
Comment (ruthhalteman) [inline src/inboxsdk.d.ts:420]:
```suggestion
   * If you want to use an icon font with ligatures enabled, you can use the `iconLiga` property.
```

Comment (wegry) [reply src/inboxsdk.d.ts]: See  5985a5d

**Review by ruthhalteman — APPROVED** [review]
